src/js/utils/rxjsUtils.js

```diff
@@ -0,0 +1,19 @@
+import { timer } from "rxjs/observable/timer";
+
+import "rxjs/add/operator/delayWhen";
+import "rxjs/add/operator/retryWhen";
+import "rxjs/add/operator/scan";
+
+/* eslint-disable import/prefer-default-export */
+export function linearBackoff(maxRetries = 5, delay = 1000, scheduler = null) {
```

Comment:
This is a really cool solution.

Have you looked at how AWS has figured out a "better way" to use exponential backoff?
https://aws.amazon.com/blogs/architecture/exponential-backoff-and-jitter/

Reply:
I haven't, that's a good read BTW, thank you! I wonder if we need this kind of backoff in this case 🤔

Reply:
I very much like the idea of using jitter to spread the requests but don't think that we need this kind of backoff strategy in this case. AWS develop this technique to improve the DynamoDB *write* performance (note that writes are blocking) when a huge number of clients tries to do so at the same time. Using a linear or exponential backoff should be sufficient,  as we only read data and don't have a huge number of clients connecting at the same time.

Reply:
Agree with @orlandohohmeier. @weblancaster this makes me think of network collision strategies like CSMA/CD (https://en.wikipedia.org/wiki/Carrier-sense_multiple_access_with_collision_detection)

Reply:
Careful @Fabs, you're giving me flashbacks to networking architecture courses I had to take in University... 😏

Reply:
Haha. Someone got the Knuth books in HH :D, and there is also a CLRS. Life is remembering 🐰